qa/L0_dyna_sequence_batcher/dyna_sequence_batcher_test.py

```diff
@@ -46,14 +46,12 @@
 
 _no_batching = (int(os.environ.get('NO_BATCHING', 0)) == 1)
 
-_trials = ("custom", "savedmodel", "graphdef", "plan", "onnx", "libtorch")
+_trials = ("savedmodel", "graphdef", "plan", "onnx", "libtorch")
 if _no_batching:
     _trials += ("savedmodel_nobatch", "graphdef_nobatch", "plan_nobatch",
                 "onnx_nobatch", "libtorch_nobatch")
 
 _ragged_batch_supported_trials = list()
-if "custom" in _trials:
```

Comment:
@GuanLuo  should we be testing TensorRT and TF (and soon ORT) here since they support ragged batching?

Reply:
[DLIS-2318](https://jirasw.nvidia.com/browse/DLIS-2318)